web/src/views/ProfileEdit.tsx

```diff
@@ -226,7 +226,8 @@ const ProfileEdit: React.FC = (props: any) => {
                     <Text as="h3" color={theme.colors.contrast} mx={2}>
                       {c.title}
                     </Text>
-                    {!profileState.hasPendingEdit && profileState.isProvisioned && (
+                    {((!profileState.hasPendingEdit && profileState.isProvisioned) ||
```

Comment:
With the revisions in #458 I think you could remove this logic entirely. Eventually, it would be good to `disable` fields from being able to be edited if their is a pending edit.

Reply:
updated